common/src/main/java/org/conscrypt/HpkeContextSender.java

```diff
@@ -16,21 +16,20 @@
 
 package org.conscrypt;
 
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.Provider;
 import java.security.PublicKey;
-import org.conscrypt.NativeRef.EVP_HPKE_CTX;
 
 /**
  * Hybrid Public Key Encryption (HPKE) Sender APIs.
  *
  * @see <a href="https://www.rfc-editor.org/rfc/rfc9180.html#hpke-export">HPKE RFC 9180</a>
  */
-public class HpkeContextSender {
-    private final HpkeSuite hpkeSuite;
-    private EVP_HPKE_CTX ctx;
-    private byte[] enc;
-
-    private HpkeContextSender(HpkeSuite hpkeSuite) {
-        this.hpkeSuite = hpkeSuite;
+class HpkeContextSender extends HpkeContext{
```

Comment:
The caller will be using these classes as `HpkeContextSender.getInstance(alg);` right? If that is the case, we should grant such visibility